packages/data/addon/models/navi-fact-response.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import EmberObject from '@ember/object';
+import { ResponseV1 } from 'navi-data/serializers/facts/interface';
+import moment, { Moment, MomentInput } from 'moment';
+import { TimeDimensionColumn } from './metadata/time-dimension';
+
+function notNull<T>(t: T | null): t is T {
+  return t !== null;
+}
+
+export default class NaviFactResponse extends EmberObject implements ResponseV1 {
+  rows!: ResponseV1['rows'];
+  meta!: ResponseV1['meta'];
```

Comment:
Maybe we can do something like this
```suggestion
  rows: ResponseV1['rows'] = [];
  meta: ResponseV1['meta'] = {};
```

Reply:
yeah, I guess we are doing this anyways in the normalizers.